rules/ethtool.mk

```diff
@@ -0,0 +1,12 @@
+# ethtool
+
+ETHTOOL_VERSION_BASE = 5.8
```

Comment:
@shlomibitton why do you need this? Where it has been used?

Reply:
I have replaced the checkout to a 'tag' and not 'hash', now it is used